server/tailwind.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     transform: {
       java: (content) => {
         var output = []
-        for (const match of content.matchAll(/"([\w-/:]+)"/g)) {
+        for (const match of content.matchAll(/"([\w-/:.]+)"/g)) {
```

Comment:
Adding the "." here is subtle, but is necessary in order to enable tailwind styles with decimal components (e.g. space-x-3.5).